packages/frontend/src/components/wallet/ExploreNativeBanner.js

```diff
@@ -45,12 +64,18 @@ const StyledBannerLink = styled(Link)`
 const ExploreNativeBanner = () => {
     return (
         <StyledContainer>
-            <img src={exploreAppsImg} alt="Explore Apps Banner" width="211" height="180"/>
-            <h2><Translate id='exploreApps.exploreNear'/></h2>
-            <div><Translate id='exploreApps.text'/></div>
-            <StyledBannerLink to="/explore">
-                <Translate id='exploreApps.exploreApps'/>
-            </StyledBannerLink>
+            <img src={exploreAppsImg} alt="Explore Apps Banner" width="211" height="180" />
+            <h2><Translate id='exploreApps.exploreNear' /></h2>
+            <div><Translate id='exploreApps.text' /></div>
+            {!SHOW_EXPLORE_PAGE ? (
```

Comment:
I think this should be `SHOW_EXPLORE_PAGE ?` without the `!`

Reply:
yes, was testing to make sure the condition worked but forgot to remove it :)